### PR TITLE
Simplify Pypy2/3 Travis setup

### PR DESCRIPTION
**Pull request by rht**
- [x] This is contingent on https://github.com/travis-ci/travis-cookbooks/pull/861.

### CODE REVIEW

**Comment by peterjc** [comment]
Have TravisCI updated the default PyPy yet, otherwise I would have expected this to fail...

**Comment by peterjc** [comment]
Ah, I see now - fingers crossed your TravisCI pull request gets accepted as I'd also like to remove this PyPy hack.

**Comment by rht** [comment]
(maybe :soon: ?) Or there should be a way to specify a hash for the cookbook (https://travis-ci.org/biopython/biopython/jobs/255329387#L23).

File: .travis.yml [hunk 1]
```diff
@@ -13,8 +13,11 @@
 # - basics - quick things like style and packaging
 # - test - the actual functional tests which are slow
 
+dist: trusty
 language: python
-cache: pip
+cache:
+  - pip
+  - apt: false
```
Comment (peterjc) [inline .travis.yml]: Why exclude apt from the cache? We install several command line tools for testing using the apt packages.

Comment (rht) [inline .travis.yml]: Oh, I was tinkering the config file to see if not using any cache could update the build image to a recent version. I should have set the commit message with "[test]".

**Comment by peterjc** [comment]
It seems TravisCI never tested the final commit. Could you try rebasing the branch, or adding yourself to the ``NEWS.rst`` and ``CONTRIB.rst`` files to trigger a fresh build please?

Also, are you happy to dual licence your contributions under both the _Biopython License Agreement_ AND the _BSD 3-Clause License_? See #898.

**Comment by peterjc** [comment]
I've just merged #1350 which also changed ``.travis.yml`` so ideally could you rebase this? If not, I'll try to do this myself later in the week. Thanks.

**Comment by peterjc** [comment]
On pull request #1351, @junaruga has confirmed these changes still work nicely on the updated master branch (as expected).

@rht Before merging this, we still need you to agree to dual licence your contributions under both the *Biopython License Agreement* AND the *BSD 3-Clause License*. See #898. Thank you.

**Comment by peterjc** [comment]
Thank you for agreeing to the licensing.

Also, are you happy to be thanked by name in the ``NEWS.rst`` and ``CONTRIB.rst`` files?

**Comment by rht** [comment]
@peterjc yes, please. I am not sure I can predict my mental state to be able to answer the question.

**Comment by peterjc** [comment]
Would you like to be listed as just ``rht`` in the ``NEWS.rst`` and ``CONTRIB.rst`` files?

(Most people give a full name, but we have at least one contributor appearing under their GitHub username only)

**Comment by rht** [comment]
Yes, I would appreciate such!

(Iff it is fine. While it is completely legal, it is impossible by choice, for instance, to contribute pseudonymously to the Linux Kernel project.)

**Pull request merged**